src/currencyEngineXMR.js

```diff
@@ -783,50 +693,68 @@ class MoneroEngine {
         throw new Error('Error invalid payment id')
       }
     }
+
+    let priority = 2
+    // Fee estimates
+    if (edgeSpendInfo.networkFeeOption) {
+      switch (edgeSpendInfo.networkFeeOption) {
+        case 'low':
+          priority = 1
+          break
+        case 'standard':
+          priority = 2
+          break
+        case 'high':
+          priority = 4
+          break
+      }
+    }
+
     let result
-    let options: SendFundsOptions
+    let sendParams: SendFundsParams
     try {
       const amountFloatString: string = bns.div(nativeAmount, '1000000000000', 8)
       // Todo: Yikes. Why does mymonero-core-js take a float, not a string? -paulvp
       const amountFloat = parseFloat(amountFloatString)
-      options = {
-        isRingCT: true,
-        target_address: publicAddress, // currency-ready wallet address, but not an OA address (resolve before calling)
-        nettype: MAINNET,
-        amount: amountFloat, // number
-        wallet__keyImage_cache: this.keyImageCache,
-        wallet__public_address: this.walletLocalData.moneroAddress,
-        wallet__private_keys: { view: this.walletLocalData.moneroViewKeyPrivate, spend: this.walletInfo.keys.moneroSpendKeyPrivate },
-        wallet__public_keys: { view: this.walletLocalData.moneroViewKeyPublic, spend: this.walletInfo.keys.moneroSpendKeyPublic },
-        hostedMoneroAPIClient: this.hostedMoneroAPIClient,
-        monero_openalias_utils: MoneroOpenAliasUtils,
-        payment_id: uniqueIdentifier,
-        mixin: 6,
-        simple_priority: 1,
-        doNotBroadcast: true
+
+      sendParams = {
+        moneroAddress: this.walletLocalData.moneroAddress,
+        moneroSpendKeyPrivate: this.walletInfo.keys.moneroSpendKeyPrivate,
+        moneroSpendKeyPublic: this.walletInfo.keys.moneroSpendKeyPublic,
+        moneroViewKeyPrivate: this.walletLocalData.moneroViewKeyPrivate,
+        targetAddress: publicAddress,
+        floatAmount: amountFloat,
+        moneroViewKeyPublic: this.walletLocalData.moneroViewKeyPublic,
+        nettype: 'mainnet', // 'mainnet' only for now
+        isSweepTx: false,
+        paymentId: uniqueIdentifier || '',
+        priority,
+        doBroadcast: false,
+        onStatus: (code: number) => {
+          console.log(`SendFunds - onStatus:${code.toString()}`)
+        }
       }
-      result = await this.SendFundsWithOptionsAsync(options)
+      result = await this.myMoneroApi.sendFunds(sendParams)
```

Comment:
Remove private key from sendParams and only add before call. {...sendParams, spendPrivateKey} @thehobbit85